src/main/java/edu/wpi/first/pathweaver/FxUtils.java

```diff
@@ -145,4 +148,19 @@ public static void loadMainScreen(Scene scene, Class aClass) {
     return new TextFormatter<>(formatter);
   }
 
+  /**
+   * Asks user if they want to delete value.
+   * @param value The file to delete
+   * @return True if user confirms deletion, false otherwise.
+   */
+  public static boolean promptDelete(String value) {
+    Alert alert = new Alert(Alert.AlertType.CONFIRMATION);
+    alert.setTitle("Delete " + value + "?");
```

Comment:
The header text also needs to be set. The default is just "Confirmation", which isn't very helpful